msprime/simulations.py

```diff
@@ -1343,3 +1344,33 @@ def print_history(self, output=sys.stdout):
             print("=" * len(s), file=output)
             self._print_populations(epoch, output)
             print(file=output)
+
+    def popn_size_history(self):
```

Comment:
We're not using the 'popn' abbreviation anywhere else, so it'll be confusing to have it here. Can you change to ``population_size_history`` please?

Reply:
Also, probably both of these functions would be better as properties (unless we can imagine passing parameters?)